region/brinstar/pink/Pink Brinstar Power Bomb Room.json

```diff
@@ -199,6 +129,82 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway",
```

Comment:
It's a little tricky because a hopper hit can throw off your positioning on the runway. It's possible to do, but depending on how the runway is used (e.g. if you need to position precisely on the edge), I'm not sure it's necessarily easier than clearing A.